orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/servergroup/CloneServerGroupTask.groovy

```diff
@@ -86,11 +86,13 @@ class CloneServerGroupTask extends AbstractCloudProviderAwareTask implements Tas
 
     List<Map<String, Object>> descriptions = []
     // NFLX bakes images in their test account. This rigmarole is to allow the prod account access to that image.
+    Collection<String> targetRegions = operation.region ? [operation.region] :
```

Comment:
Just based on the PR description, `availabilityZones` is the preferred default, so should these be flipped?